datahub-web-react/src/app/entity/dataset/shared/DatasetStatsSummary.tsx

```diff
@@ -47,6 +50,7 @@ export const DatasetStatsSummary = ({
     const statsViews = [
         !!rowCount && (
             <ExpandingStat
+                renderCss={false}
```

Comment:
really don't like that we require a new prop specifically determining whether this component applies CSS. it feels like this is a leaky abstraction.

the expanding stat container should just get larger, and then the stats inside should default to not wrapping.

Reply:
okay i will try with withount rendercss prop . actually i have conditionally render css for rowcount in exapndingstat component for datasummary page because  rowcount hiding content and  not visible so i have added specific css to it